posts/templates/posts/create_post.html

```diff
@@ -0,0 +1,27 @@
+{% load static from staticfiles %}
+{% load static %}
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Create a Post</title>
+        <link rel="stylesheet" type="text/css" href="{% static 'styles.css' %}"/>
+    </head>
+    <body>
+        <h1>Create a anonymous Post</h1>
+        <form action="{% url 'posts:create' %}" method="post">
+            {% csrf_token %}
+            Title <br>
+            {{ form.title }}<br>
+            {% if form.errors %}
```

Comment:
`form.errors` and `form.title.errors` are inconsistent. You just need the latter, by iterating through them

Reply:
i will make that change

Reply:
Fix this